packages/protocol/test/L1/TaikoL1LibProvingWithTiers.t.sol

```diff
@@ -153,43 +154,64 @@ contract TaikoL1LibProvingWithTiers is TaikoL1TestBase {
                 false
             );
 
+            // Try to contest
+            proveBlock(
+                Carol,
+                Carol,
+                meta,
+                parentHash,
+                blockHash,
+                signalRoot,
+                minTier,
+                "",
+                false
+            );
+
+            vm.roll(block.number + 15 * 12);
+
+            vm.warp(block.timestamp + L1.getTier(minTier).cooldownWindow + 1);
+
+            // Cannot verify block because it is contested..
+            verifyBlock(Carol, 1);
+
             if (minTier == LibTiers.TIER_OPTIMISTIC) {
```

Comment:
Can we extract a method for these if-elses, I saw the same patten in the same file.